pkg/controllers/disruption/metrics.go

```diff
@@ -23,17 +23,25 @@ import (
 
 func init() {
 	crmetrics.Registry.MustRegister(deprovisioningDurationHistogram, deprovisioningReplacementNodeInitializedHistogram, deprovisioningActionsPerformedCounter,
-		deprovisioningEligibleMachinesGauge, deprovisioningReplacementNodeLaunchFailedCounter, deprovisioningConsolidationTimeoutsCounter)
+		deprovisioningEligibleMachinesGauge, deprovisioningReplacementNodeLaunchFailedCounter, deprovisioningConsolidationTimeoutsCounter,
+		disruptionEvaluationDurationHistogram, disruptionReplacementNodeClaimInitializedHistogram, disruptionReplacementNodeClaimFailedCounter,
+		disruptionActionsPerformedCounter, disruptionEligibleNodesGauge, disruptionConsolidationTimeoutTotalCounter)
 }
 
 const (
 	deprovisioningSubsystem = "deprovisioning"
 	deprovisionerLabel      = "deprovisioner"
-	actionLabel             = "action"
-	consolidationType       = "consolidation_type"
+
+	disruptionSubsystem = "disruption"
+	disruptionTypeLabel = "disruption_type"
+	actionLabel         = "action"
+	consolidationType   = "consolidation_type"
```

Comment:
Thoughts on removing this consolidation_type and just calling it disruption_type? We can make the consolidation_type values for the timeout be `single-node-consolidation` etc.
```suggestion
	disruptionSubsystem = "disruption"
	disruptionTypeLabel = "disruption_type"
	actionLabel         = "action"
```

Reply:
👍